src/styles/AddProperty.css

```diff
@@ -0,0 +1,43 @@
+h1 {
```

Comment:
Indentation in the CSS is a bit inconsistent with the JS, where you have 2 spaces for the most part. It should be the same across the whole project